vmc/resource_vmc_sddc.go

```diff
@@ -383,8 +395,12 @@ func resourceSddcRead(d *schema.ResourceData, m interface{}) error {
 		d.Set("num_host", len(sddc.ResourceConfig.EsxHosts))
 		d.Set("vpc_cidr", *sddc.ResourceConfig.VpcInfo.VpcCidr)
 		d.Set("vxlan_subnet", sddc.ResourceConfig.VxlanSubnet)
+		sddcSizeInfo := map[string]string{}
+		sddcSizeInfo["vc_size"] = *sddc.ResourceConfig.SddcSize.VcSize
+		sddcSizeInfo["nsx_size"] = *sddc.ResourceConfig.SddcSize.NsxSize
+		sddcSizeInfo["size"] = *sddc.ResourceConfig.SddcSize.Size
```

Comment:
Do we need this? can it not be inferred from vc & nsx size?

Reply:
It can be but since size is returned in the ResourceConfig.SddcSize object I think we should save that too.

Reply:
ok

Reply:
Done